### PR TITLE
Removed dart:io usage, style changes

### PR DESCRIPTION
**Pull request by thosakwe**
The main thing in this PR is that `dart:io` usage was entirely removed, as it is incompatible with `package:file`. 

Also did some housekeeping, including fixing several static analysis errors, and changing the test setup so that Travis would actually run the tests.

I also renamed methods from `snake_case` to `camelCase`, as per Dart convention.

### CODE REVIEW

**Review by thislooksfun — COMMENTED** [review]
It looks good overall, but there are a few questions before I'm comfortable merging it.

File: example/layouts.dart [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
+import 'package:file/local.dart';
 
 Future main() async {
   Angel angel = new Angel();
-  await angel.configure(compiled_mustache(new Directory('views'), defaultLayout: 'home'));
-  
-  // This will render the file 'views/pages/hello.mustache'
-  // using the layout 'views/layouts/home.mustache'  (uses the default)
-  // with the context of 'name' = 'world'
-  var rendered = await angel.viewGenerator('hello', {'name': 'world'});
```
Comment (thislooksfun) [inline example/layouts.dart:12]: Why remove this example?

File: pubspec.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@ dependencies:
   angel_framework: "^1.0.0-dev"
   compiled_mustache: "^1.0.1"
   path: "^1.5.0"
-  file: "^2.3.5"
+  file: "^2.0.0"
```
Comment (thislooksfun) [inline pubspec.yaml:12]: Why move this down to 2.0.0?

**Comment by thislooksfun** [comment]
Actually, if you could re-submit this PR to the `develop` branch, not `master`, that would be appreciated. There are a couple things I want to work in before releasing it.

**Comment by thosakwe** [comment]
Will do; sorry for the late response.

**Pull request closed without merging**